### DISCUSSION TITLE
Editing in VSCode

### DISCUSSION THREAD

**Discussion by prgmr-1** [codefori/vscode-ibmi#2161] [IBM i]
in RDi I am use to using rr3 at beginning line of source and rr3 at ending line of source to shift the entire block right three spaces.  Does VSCode have an equivalent?

**Comment by SJLennon** [comment]
@prgmr-1 
`Ctrl + ]` to move one tab stop to the right. ` Ctrl + [` to move one tab stop to the left. Works on single lines or a set of highlighted lines.

**Reply by SJLennon** [reply]
For future reference, there is a link to VS Code shortcuts that may be useful:
![image](https://github.com/user-attachments/assets/7434076d-139b-4b9f-8a03-831bbbbe2de0)

**Comment by thomprl** [comment]
Also Shift-Alt-F will format the document indentions to how you have them defined.
